src/jit/Backend.cpp

```diff
@@ -935,8 +944,25 @@ void JITCompiler::generateCode()
         return;
     }
 
+#ifdef WALRUS_JITPERF
+    const bool perfEnabled = JITFlags() & JITFlagValue::perf;
+    PerfDump pd;
+    if (perfEnabled) {
+        pd.openFile();
+    }
+#endif
```

Comment:
What about merging this block with another `WALRUS_JITPERF` block (starting from line 957) below?

Reply:
One more thing,
dump file is opened by invoking `openFile` method here, but the file is implicitly closed when the destructor of `PerfDump` is called.
IMO It's better to unify file open/close policy, like explicitly calling dedicated open/close methods or implicitly open/close in constructor/destructor of `PerfDump` class.
What do you think about it?

Reply:
> What about merging this block with another `WALRUS_JITPERF` block (starting from line 957) below?

I've tried to merge it, but it didn't work, it caused a problem in `perf report`